src/Faker/Provider/fr_FR/PhoneNumber.php

```diff
@@ -116,7 +116,7 @@ public function phoneNumber08()
      */
     public function phoneNumber08WithSeparator()
     {
-        $regex = '([012]{1}\d{1}|(9[1-357-9])( \d{2}){3}';
+        $regex = '([012]{1}\d{1}|9[1-357-9])( \d{2}){3}';
```

Comment:
This needed to be changed since the test on this method tested with another regex.
The previous `array_replace(['(', ')'], '', ...)` solved this problem behind the scenes.

https://github.com/fzaninotto/Faker/blob/master/test/Faker/Provider/fr_FR/PhoneNumberTest.php#L56